float64 xmin_LiDAR
float64 xmax_LiDAR
float64 ymin_LiDAR
float64 ymax_LiDAR
float64 xcenter
float64 ycenter
float64 zcenter